_emails/newsletter-2020-02.html

```diff
@@ -0,0 +1,108 @@
+---
+title: Origami Newsletter, February 2020
+companion_post_url: https://origami.ft.com/blog/2020/02/??/newsletter/
+---
+
+<!-- TL;DR -->
+{% capture text %}
+A new component, a project board and a post-mortem.
+{% endcapture %}
+
+{% include email/tldr.html content=text %}
+
+<!-- Top things -->
+{% include email/h2.html content="Top Things" %}
+
+{% capture text %}Here's the big stuff we've done this month:{% endcapture %}
+{% include email/markdown.html content=text %}
+
+<!-- Thing 1 -->
+{% include email/h3.html content="Origami Project Board" %}
+{% capture text %}
+
+In our [continued](https://origami.ft.com/blog/2020/01/31/newsletter/) efforts
+towards openness we've created a [GitHub project
+board](https://github.com/orgs/Financial-Times/projects/83?fullscreen=true) that
+has a card for everything we're currently working on.
+
+When an issue is opened on any of our projects, it's added to the incoming
+column automatically by [a GitHub
+action](https://github.com/Financial-Times/origami-project-board-action/).
+
+We started with 348 things to do. We haven't yet seen that number drop below
+three hundred, but we've done 534 of them and we're working on the rest.
+
+![capture of Newman from Seinfeld with the caption "the mail never stops" ](https://www.ft.com/__origami/service/image/v2/images/raw/https://origami.ft.com/assets/images/2020-02-28-newsletter/the-mail-never-stops.png?source=origami)
+
+{% endcapture %} {% include email/markdown.html content=text %}
+
+<!-- Thing 2 -->
+{% include email/h3.html content="Layout post-mortem" %} {% capture text %}
+
+The latest version of Chrome made some unexpected changes to how they rendered
+things, and it broke the layout on a few of our internal projects. We reacted
+fast and got it fixed, and Lee wrote a great breakdown on how it happened and
+what we changed.
+
+Read it here: [CSS Grid in Chrome
+80](https://origami.ft.com/blog/2020/02/17/o-layout-chrome-80/)
+
+{% endcapture %} {% include email/markdown.html content=text %}
+
+<!-- Thing 3 -->
+{% include email/h3.html content="Meter component" %}
+
+{% capture text %}
+
+We've created a new component that was in high demand, the
+[Meter](https://registry.origami.ft.com/components/o-meter).
```

Comment:
Because of the way the registry works, this is a mostly-empty page, change to include internal brand:
```suggestion
[Meter](https://registry.origami.ft.com/components/o-meter?brand=internal).
```